File: POO JAVA/Aula11/src/aula11/Aluno.java
package aula11;

public class Aluno extends Pessoa{

    //Atributos da Classe
    private int matricula;
    private String curso;

    //Metodos Acessores e Modificadores
    public int getMatricula() {
        return matricula;
    }
    public void setMatricula(int matricula) {
        this.matricula = matricula;
    }
    public String getCurso() {
        return curso;
    }
    public void setCurso(String curso) {
        this.curso = curso;
    }
    
    //Metodo da Classe
    public void pagarMensalidade(){
        System.out.println("Pagando a Mensalidade do Aluno "+this.nome);
    }
    
}

File: POO PHP/Aula11/Aluno.php
<?php

require_once 'Pessoa.php';

class Aluno extends Pessoa{

    //Atributos da Classe
    private $matricula;
    private $curso;
    
    //Metodos Acessores e Modificadores
    public function getMatricula() {
        return $this->matricula;
    }
    public function getCurso() {
        return $this->curso;
    }
    public function setMatricula($matricula): void {
        $this->matricula = $matricula;
    }
    public function setCurso($curso): void {
        $this->curso = $curso;
    }

    //Metodo da Classe
    public function pagarMensalidade() {
        echo "<p>Pagando a mensalidade do(a) ".$this->getNome()."</p>";
    }
}
